pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4618,90 +4618,356 @@ func Test_server_MultiUpsert(t *testing.T) {
 
 func Test_server_Remove(t *testing.T) {
 	t.Parallel()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
 	type args struct {
-		ctx context.Context
-		req *payload.Remove_Request
-	}
-	type fields struct {
-		name              string
-		ip                string
-		ngt               service.NGT
-		eg                errgroup.Group
-		streamConcurrency int
+		ctx      context.Context
+		indexId  string
+		removeId string
```

Comment:
```suggestion
		originId  string
		indexedId string
```
or
```suggestion
		indexId  string
		targetId string
```
umm... :thinking: 
Of course, I do not care if you won't change.

Reply:
My intent is that `indexId` means the id which is in the Vald and `removeId` means  the id which is request for remove.
It is too difficult so let me put it in current expression for once.